python/cuml/cuml/svm/svm_base.pyx

```diff
@@ -373,13 +384,36 @@ class SVMBase(Base,
         cdef lib.SvmParameter param
         param.C = self.C
         param.cache_size = self.cache_size
-        param.max_iter = self.max_iter
         param.nochange_steps = self.nochange_steps
         param.tol = self.tol
         param.verbosity = self.verbose
         param.epsilon = self.epsilon
         param.svmType = lib.SvmType.C_SVC if is_classifier else lib.SvmType.EPSILON_SVR
 
+        if isinstance(self.max_iter, TotalIters):
+            param.max_iter = int(self.max_iter)
+            param.max_outer_iter = -1
+        elif self.max_iter == -1:
+            param.max_iter = -1
+            param.max_outer_iter = -1
+        else:
+            name = type(self).__name__
+            warnings.warn(
+                (
+                    "The meaning of `max_iter` will change in version 26.02 from "
+                    "'max outer iterations' to 'max total iterations'.\n\n"
+                    "You may opt into the new behavior now with:\n\n"
+                    f"  {name}(max_iter={name}.TotalIters(max_total_iter), ...)\n\n"
```

Comment:
We could be even more concrete here:
```suggestion
                    f"  {name}(max_iter={name}.TotalIters({self.max_iter_}), ...)\n\n"
```

Reply:
Originally I wrote it like that, but then realized the user will want to increase the limit from what they had before.